addons/website_event_track/static/src/js/website_event_track_proposal_add_tags.js

```diff
@@ -0,0 +1,106 @@
+odoo.define('website_event_track.website_event_track_proposal_add_tags', function (require) {
+    'use strict';
+
+    var core = require('web.core');
+    var publicWidget = require('web.public.widget');
+
+    var _t = core._t;
+
+    publicWidget.registry.websiteEventProposalTags = publicWidget.Widget.extend({
```

Comment:
Same remark: naming to update accordingly with chosen name of other widget and propagate

Reply:
Forgot, done in next version